powersimdata/scenario/execute.py

```diff
@@ -331,22 +331,22 @@ def __init__(self, data_access, scenario_info, grid, ct):
         self._scenario_info = scenario_info
         self.grid = grid
         self.ct = ct
-        self.server_config = server_setup.PathConfig(server_setup.DATA_ROOT_DIR)
-        self.scenario_folder = "scenario_%s" % scenario_info["id"]
+        self.scenario_id = scenario_info["id"]
 
-        self.REL_TMP_DIR = posixpath.join(
-            server_setup.EXECUTE_DIR, self.scenario_folder
+        self.REL_TMP_DIR = self._data_access.join(
+            server_setup.EXECUTE_DIR, f"scenario_{self.scenario_id}"
         )
+        self.TMP_DIR = self._data_access.match_scenario_files(self.scenario_id, "tmp")
 
     def create_folder(self):
         """Creates folder on server that will enclose simulation inputs."""
         print("--> Creating temporary folder on server for simulation inputs")
```

Comment:
I guess the the temporary folder can be crated either on the local machine or the server. Perhaps we can the `description` attribute here too.